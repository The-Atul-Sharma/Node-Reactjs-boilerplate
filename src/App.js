// vendor modules
import React from 'react';

export default function App() {
  return (
    <div>
      <h1>React Boilerplate</h1>
      <p>Start coding!</p>
    </div>
  );
}
