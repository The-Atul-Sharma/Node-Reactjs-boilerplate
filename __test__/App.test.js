// vendor modules
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
// react modules
import App from '../src/App';

describe('App component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<App />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('app ui changes', () => {
    const AppComponent = renderer.create(<App />).toJSON();
    expect(AppComponent).toMatchSnapshot();
  });
});
