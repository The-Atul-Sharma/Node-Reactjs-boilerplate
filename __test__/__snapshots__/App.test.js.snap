// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component app ui changes 1`] = `
<div>
  <h1>
    React Boilerplate
  </h1>
  <p>
    Start coding!
  </p>
</div>
`;
